lib/config.ts

```diff
@@ -9,16 +9,28 @@ type InitConsent = {
 };
 
 type InitConfig = {
-  host: string;
+  // Source slug for SDK calls
   site: string;
+  // API host
+  host: string;
+  // Node for API calls (only set if required by the host)
+  node?: string;
+  // Enable cookie storage
   cookies?: boolean;
+  // Previous Host, used only when switching hosts to retain previous cache state
+  legacyHostCache?: string;
+  // Initialize passport on SDK load
   initPassport?: boolean;
+  // Consent settings
   consent?: InitConsent;
+  // Enable read-only mode
   readOnly?: boolean;
 };
 
-type ResolvedConfig = Required<Omit<InitConfig, "consent">> & {
+type ResolvedConfig = Required<Omit<InitConfig, "consent" | "node" | "legacyHostCache">> & {
```

Comment:
At this point it would be clearer to define this type explicitly. Not blocker though